src/BlockNumber.jsx

```diff
@@ -13,12 +13,16 @@ export default function BlockNumber(props) {
     : api.derive.chain.bestNumber;
 
   useEffect(() => {
-    let unsub = bestNumber(number => {
+    let unsubscribeAll = bestNumber(number => {
```

Comment:
This is still not 100% right. You need to declare unsub (with `let unsub`). and then just call the subscription (`bestNumer(...`). The unsub will be set to the unsubscribe function (this you do it right).

```suggestion
    let unsubscribeAll;
    bestNumber(number => {
```

Reply:
also no need to be called unsubscribe*All*, there's only 1 subscription here unlike in the balance module where there's a subscription per account (because of the `multi`)